### ISSUE TITLE
Feature req : a sitemap plugin 

### ISSUE THREAD

**Issue by oblab** [shaarli/Shaarli#1742]
I'm not a php developper, but it would be useful to have a plugin that generate a sitemap.xml for Shaarli.
I saw this attempt https://github.com/sebsauvage/Shaarli/pull/200/files
Any idea to do this on Shaarli 0.12.1 ?

**Comment by ArthurHoaro** [comment]
Good idea. I think it would even make sense as a core feature and not as a plugin, because sitemaps widely used to index an website and pretty simple.

> Any idea to do this on Shaarli 0.12.1 ?

For now besides writing a custom script (e.g. directly manipulating the datastore or through the REST API), this is not possible.

**Comment by oblab** [comment]
In the meanwhile, I found this sitemap generator https://github.com/tristangoossens/php-sitemap-generator
It's very useful, you can add absolute links or keywords for the pages to skip when crawling. 

I just don't understand why the crawler find duplicate urls for permalink, example :
https://mydomain.com/shaare/O0PLRQ and https://mydomain.com/?O0PLRQ

**Comment by oblab** [comment]
It's in includes.html file, there is a meta property : `<meta property="og:url" content="{$index_url}?{$link.shorturl}" />`
Shouldn't this be : `<meta property="og:url" content="{$index_url}shaare/{$link.shorturl}" />` ?
